src/Apps/serverCacheTTLs.tsx

```diff
@@ -0,0 +1,21 @@
+const NO_CACHE = 0
+const HOURS_24 = 86400000
+
+/**
+ * This defines non-default cache TTLs for specific routes. Default is typically
+ * 1 hour, is set via GRAPHQL_CACHE_TTL env var, and automatically applies to
+ * all routes not defined here.
+ */
+export const serverCacheTTLs = {
+  artist: HOURS_24,
+  artists: HOURS_24,
+  artistSeries: HOURS_24,
+  auction: NO_CACHE,
+  categories: HOURS_24,
+  collect: HOURS_24,
+  collection: HOURS_24,
+  collections: HOURS_24,
+  gene: HOURS_24,
+  partners: HOURS_24,
```

Comment:
Some thoughts:

- Partners pay attentions to their listing pages (e.g. partners, partner) and I assume we'd get questions from partners if updates take max 1 day to be reflected on the site. _I think_ it's a relatively lower traffic route, so maybe it's safer to use a shorter/more reasonable TTL. Also, it's worth giving the galleries team a heads-up about the caching behavior so they can be prepared [when getting questions](https://artsy.slack.com/archives/C03N12SR0RK/p1722867890398409?thread_ts=1722861744.869819&cid=C03N12SR0RK).
- Not specific to this route, but for timely information (such as the urgency signals that Emerald is working on), updates would take max 1 day to be reflected. Not sure how significant the impact is to conversion, but I'll surface to our team to at least be aware of it.

Reply:
Once the numbers stabilize we'll make a wider announcement! PDDE Core has already been informed and we as engineers should keep an eye on things. We're still getting a very low % cache hit rate.

Reply:
But yes, that's a good point regarding partners being lower traffic, higher visibility (and not necessarily from collectors). I'll remove this entry so as to reduce the potential for confusion 👍